.github/workflows/conda.yml

```
@@ -6,7 +6,7 @@ on:
   pull_request:
     paths:
       - setup.py
-      - requirements.txt
+      - setup.cfg
       - continuous_integration/recipes/**
       - .github/workflows/conda.yml
```

Should we also check `pyproject.toml`?

```suggestion
      - pyproject.toml
```